src/renderer/api/endpoints/pods.api.ts

```diff
@@ -342,7 +342,7 @@ export class Pod extends WorkloadKubeObject {
   // Returns pod phase or container error if occurred
   getStatusMessage() {
     if (this.getReason() === PodStatus.EVICTED) return "Evicted";
-    if (this.getStatus() === PodStatus.RUNNING && this.metadata.deletionTimestamp) return "Terminating";
+    if (this.metadata.deletionTimestamp) return "Terminating";
```

Comment:
with this it will probably never show "Terminated", which is checked for below.

Reply:
Even though we use the same root word ("terminate"), these two are actually different. This means that the pod has been chosen to be stopped by its parent deployment whereas the `state.terminated` means that the pod has run and then stopped. Which is why there is an `exitCode` field under `IPodContainerStatus.state.terminated`

Reply:
Ah, thanks. So when stopped by the parent deployment we just show "Terminating" until the pod disappears from the list?

Reply:
Yes, because that is what `kubectl` does.